src/Core/Framework/DataAbstractionLayer/FieldSerializer/EnumFieldSerializer.php

```diff
@@ -48,8 +49,9 @@ public function encode(
     public function decode(Field $field, mixed $value): ?\BackedEnum
     {
         $field = $this->checkFieldTypeOrThrowInvalidFieldException($field);
+        $fieldValue = $this->castValueToFieldTypeValue($field, $value);
```

Comment:
I guess we can early return after that if `fieldValue` is null, right? would make it more readable imo, instead of the ternary operator